### PR TITLE
Landing page with featured categories

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
# Landing page with featured categories

### CODE REVIEW

File: src/components/aside-bar/AsideBar.jsx [hunk 1]
```diff
@@ -63,9 +71,13 @@ const AsideBar = () => {
...
                 <input
                   type="checkbox"
+                  checked={pants}
                   id="pants"
                   onClick={(event) =>
-                    productPageDispatch({ type: "PANTS", payload: event.target.checked })
+                    productPageDispatch({
+                      type: "PANTS",
+                      payload: event.target.checked,
+                    })
```
Comment (nikhiljugale007) [inline src/components/aside-bar/AsideBar.jsx:80]: Suggestion: you can create re-usable component for this, as you are repeating same code and changing values.

**Review by nikhiljugale007 — COMMENTED** [review]
Added one suggestion in code.
Ui is good. Good to merge.

**Review by Naman-Saxena1 — APPROVED** [review]
This is a small PR, code looks fine to me 👍

File: src/pages/landing-page/LandingPage.jsx [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+  useEffect(() => {
+    try {
+      (async () => {
+        const response = await apiCall("get", "/api/categories");
+        if (response.status === 200) {
+          setCategoryList(response.data.categories);
+        }
+      })();
+    } catch (error) {
+      console.error(error);
```
Comment (Sahil25061999) [inline src/pages/landing-page/LandingPage.jsx:19]: I think this can be removed, not sure though.

**Review by Sahil25061999 — APPROVED** [review]
Looks good, all the functionality is working, no suggestion for now.

File: src/components/aside-bar/AsideBar.jsx (hunk 1 above)
Comment (yaswanthmyneni) [reply src/components/aside-bar/AsideBar.jsx:80]: i will do it later

**Pull request merged**